Redo "badge app 101" instructions (#324)
emfcamp/badge-2024-documentation: docs/tildagon-apps/run-on-badge.md

You can test your app on-device, without publishing it, using [`mpremote`](https://docs.micropython.org/en/latest/reference/mpremote.html).

1. Plug the badge into your computer over USB-C. Make sure the cable is going into the USB IN port not the USB OUT port!
1. Install [Python](https://www.python.org/downloads/) if you don't already have it.

> 1. Install `mpremote` following the [installation instructions](https://docs.micropython.org/en/latest/reference/mpremote.html).

Reviewer: ```suggestion
1. [Install `mpremote`](https://docs.micropython.org/en/latest/reference/mpremote.html).
```

> 1. You should now be able to run `mpremote --version` in a terminal and it shouldn't error. If you don't, try `python -m mpremote --version`. If that works but `mpremote --version` didn't, you'll need to remember to type `python -m` before `mpremote` for the rest of this guide.

Reviewer: ```suggestion
1. Run `mpremote --version` in a terminal. If you do not see a version, try `python -m mpremote --version`. If that works but `mpremote --version` didn't, you'll need to remember to type `python -m` before `mpremote` for the rest of this guide.
```
